.github/workflows/prod-deploy-web.yml

```diff
@@ -12,8 +12,7 @@ jobs:
   test_web:
     uses: ./.github/workflows/reusable-web-e2e.yml
     with:
-      submodules: true
-      submodule_branch: 'main'
+      ee: true
```

Comment:
we should probably start building the enterprise docker image

Reply:
i think i will need some help related to the ee image, i cant find the old implementation in git.

Reply:
yeah, we never had ee image for the web, but now with the translation management project we would need to I guess :)